src/components/DropzoneArea.js

```diff
@@ -387,6 +387,9 @@ DropzoneArea.defaultProps = {
     initialFiles: [],
     getFileLimitExceedMessage: (filesLimit) => (`Maximum allowed number of files exceeded. Only ${filesLimit} allowed`),
     getFileAddedMessage: (fileName) => (`File ${fileName} successfully added.`),
+    getPreviewIcon: (fileObject, classes) => (isImage(fileObject.file)) ?
```

Comment:
To avoid the lint warning you reported move this to a separated function (like `defaultGetPreviewIcon`) at the top of the file (below `defaultSnackbarAnchorOrigin`) and avoid using a ternary operator, write it explicitly

```jsx
const defaultGetPreviewIcon = (fileObject, classes) => {
    if (isImage(fileObject.file)) {
        return (<img
            className={classes.smallPreviewImg}
            role="presentation"
            src={fileObject.data}
        />);
    }

    return <AttachFile className={classes.smallPreviewImg} />;
};
```

Reply:
Oh right yeah! Great idea can't believe I didn't think of that 😀